benchmark/benchmark.c

```diff
@@ -60,35 +60,34 @@ event_id_t benchmarking_events[] = {
 };
 
 #ifdef CONFIG_BENCHMARK_TRACK_UTILISATION
-static void
-microkit_benchmark_start(void)
+static void microkit_benchmark_start(void)
 {
     seL4_BenchmarkResetThreadUtilisation(TCB_CAP);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_ETH_ID);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_MUX_RX_ID);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_MUX_TX_ID);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_COPY_ID);
+    seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_COPY1_ID);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_LWIP_ID);
+    seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_LWIP1_ID);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_ARP_ID);
     seL4_BenchmarkResetThreadUtilisation(BASE_TCB_CAP + PD_TIMER_ID);
     seL4_BenchmarkResetLog();
 }
 
-static void
-microkit_benchmark_stop(uint64_t *total, uint64_t* idle, uint64_t *kernel, uint64_t *entries)
+static void microkit_benchmark_stop(uint64_t *total, uint64_t* number_schedules, uint64_t *kernel, uint64_t *entries)
 {
     seL4_BenchmarkFinalizeLog();
     seL4_BenchmarkGetThreadUtilisation(TCB_CAP);
     uint64_t *buffer = (uint64_t *)&seL4_GetIPCBuffer()->msg[0];
 
     *total = buffer[BENCHMARK_TOTAL_UTILISATION];
-    *idle = buffer[BENCHMARK_IDLE_LOCALCPU_UTILISATION];
```

Comment:
Was there are reason we don't track idle time anymore? Just curious

Reply:
Yes. This tracks the kernel idle CPU utilisation (i.e. unused cycles) which is always 0 in this system since we have our own spinning protection domain idle.c which runs when other protection domains aren't running.

Reply:
That's right, my bad I forgot about that.